modules/Microsoft.Authorization/roleAssignments/.test/mg/deploy.test.bicep

```diff
@@ -0,0 +1,45 @@
+targetScope = 'managementGroup'
+
+// ========== //
+// Parameters //
+// ========== //
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'ms.authorization.roleassignments-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param location string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'aramg'
```

Comment:
aramgcom?
Also maybe rename the folder to be `mg.common`? This comment would apply to all scopes.

Reply:
done